ksqldb-engine/src/test/java/io/confluent/ksql/statement/SourcePropertyInjectorTest.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright 2022 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
```

Comment:
I had to add this since otherwise findbugs was complaining about the `Mockito.verify()` lines in the two tests. I have no idea why this was happening -- AFAICT our other Mockito tests don't hit this issue when using `verify()`.